sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/collectionOperations.scala

```diff
@@ -1018,7 +1019,7 @@ case class Shuffle(child: Expression, randomSeed: Option[Long] = None)
       > SELECT _FUNC_(array(2, 1, 4, 3));
        [3,4,1,2]
   """,
-  group = "array_funcs",
+  group = "collection_funcs",
```

Comment:
Sorry bug this is not `Adds a group tag`. This is re-categorization.

Reply:
Ah, yea, you're right. I'll update the PR and the description.